app/Http/Controllers/Admin/AppEvaluationController.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace App\Http\Controllers\admin;
+
+use App\AppCycle;
+use App\Application;
+use App\Event;
+
+class AppEvaluationController extends Controller
+{
+    public function __construct()
+    {
+        $this->middleware('is_admin')->only([
+            'approveFeedback',
+            'createCycle',
+            'storeCycle',
+        ]);
+
+        $this->middleware('is_admin_or_manager');
+    }
+
+    public function approveFeedback($id)
+    {
+        $app = Application::find($id);
+
+        $app->update(['approved' => true]);
+
+        Event::log("Approved feedback for {$app->user->username}");
+
+        return;
+    }
+
+    public function createCycle()
+    {
+        return view('admin.app-eval.create-cycle');
+    }
+
+    public function index()
+    {
+        $cycles = AppCycle::with('applications')->get();
+
+        return view('admin.app-eval.index')
+            ->with('cycles', $cycles);
+    }
+
+    public function show($id)
+    {
+        $cycle = AppCycle::with('applications')
+            ->with('applications.answers')
+            ->with('applications.answers.question')
+            ->with('applications.feedbackAuthor')
+            ->with('applications.user')
+            ->find($id);
+
+        $apps = $cycle->applications;
+
+        $appsCollection = fractal_transform($apps, 'ApplicationTransformer');
+
+        return view('admin.app-eval.show')
+            ->with('appsCollection', $appsCollection)
+            ->with('cycle', $cycle);
+    }
+
+    public function storeCycle()
```

Comment:
i dont really get the point if the new cycle isn't active by default and marks all the others as inactive. you can't make this work without editing db unless i missed somethign?